src/librustc_ty/ty.rs

```diff
@@ -85,7 +85,7 @@ fn associated_item_from_trait_item_ref(
     let def_id = tcx.hir().local_def_id(trait_item_ref.id.hir_id);
     let (kind, has_self) = match trait_item_ref.kind {
         hir::AssocItemKind::Const => (ty::AssocKind::Const, false),
-        hir::AssocItemKind::Method { has_self } => (ty::AssocKind::Method, has_self),
+        hir::AssocItemKind::Method { has_self } => (ty::AssocKind::Fn, has_self),
```

Comment:
`hir::AssocItemKind` also seems to still be using `Method`.

Reply:
Renamed.